src/Ucommerce.Sitecore.Web/Css/Sitecore8/UI.css

```diff
@@ -123,9 +123,14 @@ body * {
   height: 100%;
 }
 
+.imagePickerContainer .imagePickerContainerBody.propertyItem {
+  display: flex !important;

```

Comment:
Is **!important** the only possible solution? 😟

Reply:
I double checked this, and the !important rule wasn't neccesery, so I've removed it :)